app/build.gradle

```diff
@@ -330,22 +332,61 @@ task chooseReleaseNotes(dependsOn: [':common:getVersion']) {
 distTar {
     dependsOn ':common:getVersion'
     compression = Compression.GZIP
-    baseName = 'crate'
     extension = 'tar.gz'
 }
-distTar.dependsOn(chooseReleaseNotes)
```

Comment:
this differs from the original commit https://github.com/crate/crate/pull/8319/files#diff-39e7d8c00954e920b98e7636f0ac30b2R330. why that?

Reply:
Because there was a conflict, therefore mergify could not do it automatically. I cherry picked it and merged it manually.

Reply:
I was more interested about why the logic changed, like why not use `tasks.withType(Tar){..}` here as well?

Reply:
oh i see, you are right, this is wrongly merged. thanks!

Reply:
It should be merged correctly now.